app/client/packages/git/src/store/gitApplicationSlice.ts

```
@@ -0,0 +1,70 @@
+/* eslint-disable padding-line-between-statements */
+import { createSlice, type PayloadAction } from "@reduxjs/toolkit";
+
+// Define a type for the slice state
+interface GitMetadata {}
```

Biome

[error] An empty interface is equivalent to {}.

Safe fix: Use a type alias instead.


(lint/suspicious/noEmptyInterface)